### ISSUE TITLE
Setting anchor box sizes for FPN in Mask RCNN

### ISSUE THREAD

**Issue by roserustowicz** [tensorflow/models#9045]
## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/official/vision/detection/dataloader/anchor.py

## 2. Describe the feature you request

In other Mask RCNN repositories (like Detectron2), you can specify the desired anchor box sizes in absolute values of the full input image size, as well as aspect ratios to apply to each of the desired anchor box sizes. In this repository, I'm confused on how to generate anchor boxes given the `anchor` configuration parameters. The configuration parameters are `anchor_size`, `aspect_ratios`, and `num_scales`. I *want* anchor boxes of sizes [8, 16, 32, 64, and 128] on the **full-sized** image, which would then have smaller corresponding sizes for the other feature pyramid layers. To try and accomplish this, I set the `max_level` to 7 to figure out what size boxes I would obtain on the full size image as I change these configuration parameters. Unfortunately the `num_scales` has a set way of setting the scale, as opposed to specifying the absolute value of each scale, so it isn't possible to set the anchor box sizes that I want. Having a way to specify absolute anchor box sizes would be ideal.

## 3. Additional context

--

## 4. Are you willing to contribute it? (Yes or No)

Yes, but I'm unsure what I would need to do to contribute.

**Comment by HUI11126** [comment]
Hello, my friend. Have solved this issue?

**Comment by gadagashwini** [comment]
Hi @roserustowicz, Take a look at workaround mentioned on similar issue [here](https://stackoverflow.com/a/69103342/14290244). Thank you!
